iac-extensions/terraform/src/test/resources/checks/GCP/HighPrivilegedRolesOnWorkloadResourcesCheck/roles_in_lists.tf

```diff
@@ -0,0 +1,63 @@
+resource "google_storage_bucket_acl" "rspecS6400_example1" {
+  bucket = google_storage_bucket.rspecS6400_example.name1
```

Comment:
Could be removed because it is not relevant for the check. The same applies to the following repetitions.